src/partition.jl

```diff
@@ -281,10 +281,19 @@ graph from the results of `partition`.
 A graph partitioning backend such as Metis or KaHyPar needs to be installed for this function to work
 if the subgraph vertices aren't specified explicitly.
 """
-function partition(
-  g::AbstractGraph; npartitions=nothing, nvertices_per_partition=nothing, kwargs...
-)
-  return partition(g, subgraph_vertices(g; npartitions, nvertices_per_partition, kwargs...))
+function partition(g::AbstractGraph; npartitions = nothing, nvertices_per_partition = nothing, sgraph_vertices = nothing, kwargs...)
```

Comment:
Better to call it `subgraph_vertices`. If it conflicts with the function call `subgraph_vertices(...)` inside the function, you can use `ITensorNetworks.subgraph_vertices(...)` to call the function.

Reply:
Ah I see, yes that is why I didn't call it that but that is a better way round!